hack/ci/deploy.sh

```diff
@@ -117,6 +117,15 @@ logging)
   deploy "promtail" "logging" charts/logging/promtail/
   ;;
 
+usercluster-mla)
+  echodate "Running Kubermatic Installer for UserCluster MLA..."
+  ./_build/kubermatic-installer deploy usercluster-mla \
+    --config "$KUBERMATIC_CONFIG" \
+    --helm-values "$VALUES_FILE" \
+    --mla-include-iap \
+    --timeout=1500s
```

Comment:
Can we please use a more descriptive timeout, like "1500000000000us"? Or what about simply "25m"?

Reply:
it's just `1500000000us` - your proposal would take much too long IMO